### PR TITLE
Fix sample code

### PR DESCRIPTION
**Pull request by bredmond-sf**

### CODE REVIEW

File: packages/generator/README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ To use an SDK client, instantiate an object of that client and configure these p
  */
 ​
 // Import the SDK in TypeScript
+// tsc requires the --esModuleInterop flag for this
 import * as CommerceSdk from "commerce-sdk";
 // For Javascript, use:
 // import as CommerceSdk from "commerce-sdk";
```
Comment (wjhsf) [inline packages/generator/README.md:43]: Missing `*` on this line.

File: packages/generator/README.md [hunk 2]
```diff
@@ -45,8 +46,9 @@ const { ClientConfig, helpers, Search } = CommerceSdk;
 // const { ClientConfig, helpers, Search } = require("commerce-sdk");
 
 // Create a configuration to use when creating API clients
-// In TypeScript, let config = new ClientConfig();
-const config = {
+let config = new ClientConfig();
```
Comment (wjhsf) [inline packages/generator/README.md:49]: Why is the `ClientConfig` instance immediately discarded?

File: packages/generator/README.md (hunk 1 above)
Comment (bredmond-sf) [reply packages/generator/README.md]: Fixed

File: packages/generator/README.md (hunk 2 above)
Comment (bredmond-sf) [reply packages/generator/README.md]: Fixed

**Pull request merged**